README.MARCJOSEPH.md

```diff
@@ -0,0 +1,32 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+2. Passengers are limited to Mom and/or Dad. No other passengers (including siblings) are currently allowed. Exceptions require permission for each occurrence.
+3.	The car must be kept clean, no trash or food will be present upon exiting the vehicle.
+
+
+## Gas Maintenance and Fueling
+maintenance considerations
+
+1. Gas will be provided only for trips to and from school,church, work or job interviews; any personal outings are your financial responsibility. For personal outings not related to schooling or the above, fuel usage must be calculated in advance and paid for by you. Parents will contribute $100 for maintenance every two months and $60 for fuel weekly, the car may not be driven unless it meets maintenance standards. The car should never be brought home with less than 1/4 of a tank. Insurance for the car is your responsibility.
+### Scheduling 	
+blurb
+
+1.	Your schedule must be edited and tuned for the day before using the car.
+## Revocation requirements 
+1.	Failure to follow the rules may result in revocation of vehicle usage privileges.
+2.	Vehicle usage privileges may be revoked based on a three-strike system:
+  - The first strike will result in the removal of personal outings, such as visiting friends.
+  - The second strike will result in the removal of food-related outings.
+  - The third strike serves as a final warning.
+    Any additional strikes will lead to permanent revocation of vehicle usage. Strikes will be issued if any rule is violated, and they reset monthly.
+## Ownership 
+1.	Any items in the car that belong to you, and are used, will remain your property. If the vehicle is revoked, any and all items inside the car that are yours must be returned or compensated for accordingly.
```

Comment:
add a line that states - "Ultimate ownership of the car is currently upon the parents, if parents need the vehicle they will always get priority"

Reply:
have we stated anywhere the funds that will be made available each month for gas and upkeep?